Library/Formula/vault.rb

```diff
@@ -0,0 +1,68 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  url "https://github.com/hashicorp/vault.git",
+      :tag => "v0.1.1",
+      :revision => "074dcaadac2e0a46ae28f6d491885165b7431aae"
```

Comment:
No need for both `:tag` and `:revision` - `:tag` is enough to tell Homebrew to fetch the repository at the commit referenced by that tag.